testApi/src/main/scala/africa/absa/testing/scapi/rest/response/Response.scala

```diff
@@ -80,15 +81,15 @@ object Response {
       Logger.debug(s"Response-${resolvedResponseAction.group}: '${resolvedResponseAction.name}' - error details:$baseLog$exceptionLog")
     }
 
-    responseAction.iterator.map { assertion =>
-      val resolvedResponseAction: ResponseAction = assertion.resolveByRuntimeCache()
+    responseActions.iterator.map { responseAction =>
```

Comment:
Why not just

```suggestion
    responseActions.map { responseAction =>
```

Reply:
Fixed in commit a446cfff190bbb995b33cfc8b677cfba6bff48cd.